taggit/managers.py

```diff
@@ -203,9 +206,31 @@ def add(self, *tags):
         for new_tag in tags_to_create:
             tag_objs.add(self.through.tag_model().objects.create(name=new_tag))
 
+        new_ids = set(t.pk for t in tag_objs)
+
+        # NOTE: can we hardcode 'tag_id' here or should the column name be got
```

Comment:
The `TaggableManagerOfficialTestCase` passes and the [custom tag docs](http://django-taggit.readthedocs.io/en/latest/custom_tagging.html#custom-tag) suggest so but a second opinion would be good.